src/utils/sort.ts

```diff
@@ -0,0 +1,28 @@
+import { TableCellHeaderArrowDirection } from "../components/TableCellHeader";
+import { Sort } from "../types";
+import { OrderDirection } from "../types/globalTypes";
+
+export function getSortUrlVariables<TSortKey extends string>(
+  field: TSortKey,
+  params: Sort<TSortKey>
+): Sort<TSortKey> {
+  if (field === params.sort) {
+    return {
+      asc: !params.asc,
+      sort: field
+    };
+  }
+
+  return {
+    asc: true,
+    sort: field
+  };
+}
+
+export function getOrderDirection(asc: boolean): OrderDirection {
+  return asc ? OrderDirection.ASC : OrderDirection.DESC;
+}
+
+export function getArrowDirection(asc: boolean): TableCellHeaderArrowDirection {
+  return asc ? "asc" : "desc";
```

Comment:
You should use `OrderDirection enum`

Reply:
`TableCellHeader` will eventually end up in Macaw, so it's necessary to keep UI and API separated.